src/driver/Driver.ts

```diff
@@ -132,7 +132,7 @@ export interface Driver {
     /**
      * Prepares given value to a value to be persisted, based on its column type.
      */
-    prepareHydratedValue(value: any, column: ColumnMetadata): any;
+    prepareHydratedValue(value: any, column: ColumnMetadata | string): any;
```

Comment:
I guess that would at least match the overload for both cases without interfering with existing code.
```suggestion
    prepareHydratedValue(value: any, column: ColumnMetadata): any;
    prepareHydratedValue(value: any, column: string): any;
    prepareHydratedValue(value: any, column: ColumnMetadata | string): any;
```